algorithms/scripts/SGDClassifier-Isomap-KT12.py

```diff
@@ -3,7 +3,7 @@
 
 # # Create a logistic regression model to predict TP53 mutation from gene expression data in TCGA using Isomap as a dimensionality reduction technique
 
-# In[1]:
+# In[27]:
```

Comment:
Ah the diff would be way nicer with a freshly run notebook (so cells start at 1). Is is feasible to restart and run all cells, so this part of the diff goes away?